Makefile

```diff
@@ -0,0 +1,36 @@
+include ./Makefile.os
+include ./Makefile.docker
+include ./Makefile.maven
+
+PROJECT_NAME ?= kafka-bridge
+GITHUB_VERSION ?= master
+RELEASE_VERSION ?= latest
+
+ifneq ($(RELEASE_VERSION),latest)
+  GITHUB_VERSION = $(RELEASE_VERSION)
+endif
+
+all: java_package docker_build docker_push
+clean: java_clean
+release: release_prepare release_maven release_package
+
+next_version:
```

Comment:
what's this next_version  and the related `NEXT_VERSION` env var?

Reply:
This is for setting the new version in master after a release ... e.g. to bump it from 0.12.0-SNAPSHOT to 0.13.0-SNAPSHOT.

Reply:
Thanks! I am going to merge.